lib/utils/domFns.js

```diff
@@ -109,13 +109,28 @@ export function offsetXYFromParent(evt: {clientX: number, clientY: number}, offs
   return {x, y};
 }
 
-export function createCSSTransform({x, y}: {x: number, y: number}): Object {
-  // Replace unitless items with px
-  return {[browserPrefixToKey('transform', browserPrefix)]: 'translate(' + x + 'px,' + y + 'px)'};
+export function createCSSTransform({x, y}: ControlPosition, positionOffset: PositionOffsetControlPosition): Object {
+  let translation;
+  if (positionOffset && (positionOffset.x !== 0 || positionOffset.y !== 0)) {
```

Comment:
Any reason to check for 0 here - it will still work w/o it and make the code a bit simpler.

Reply:
I was thinking we would need the transform if 0 was passed, but I guess in that case the transform doesn't do anything. I'll remove it.

Reply:
Done